frontend/src/hooks/useZaakSelection.test.ts

```diff
@@ -2,7 +2,6 @@ import { act, renderHook, waitFor } from "@testing-library/react";
 
 import { ZaakSelectionContextProvider } from "../contexts";
 import { ZaakIdentifier } from "../lib/zaakSelection";
```

Comment:
the eslint imports plugin has options to forbid parent-path imports, so that you have to use the 'absolute' paths. Obviously haven't enabled it because that would mess around with your code a lot, and I don't know whether you folks consider this a problem or not :)

Reply:
No strong opinions here. I kinda landed into this approach because of the new builder each day sentiment in frontend tooling. I want te rely as little as possible on features provided by build tooling for flexibility. (I still stand behind this principle btw.). So aliases were avoided.

Also IIRC CRA had some issues with certain aliases not working correctly so I avoided them.

As far a I can tell (almost) every build toolchain provides proper aliases now and I don't mind introducing them (knowingly that the import/file) structure has seen some changes in the path so I fresh look it wouldn't hurt.

Reply:
Yeah, it was more of a head's up that you can now enforce a consistent style if you want, I think it's a very sensible thing to let the project lead decide which direction to take!